sdk/core/azure-core/src/main/java/com/azure/core/util/polling/DefaultSyncPoller.java

```diff
@@ -62,7 +62,7 @@
             throw logger.logExceptionAsWarning(new IllegalArgumentException(
                 "Negative or zero value for 'defaultPollInterval' is not allowed."));
         }
-        this.defaultPollInterval = pollInterval;
+        this.pollInterval = pollInterval;
```

Comment:
just noticed that the existing error message has a typo: `..value for 'defaultPollInterval' is not` -> `..value for 'pollInterval' is not`

(also in other places we throw this error)

Reply:
Fixed.